UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs

```diff
@@ -372,7 +380,7 @@ public void CheckWindOtherPush()
 
 			if (Data.magnitude > 0.1f)
 			{
-				if (PushingData != Data)
+				if (PushingData != Data && Data.magnitude > (int)WindStrength.WEAK)
```

Comment:
8 tiles per second is pretty damn fast, that's the same as player movement pretty much

Reply:
> 8 tiles per second is pretty damn fast, that's the same as player movement pretty much

All wind push movements I've tested have been very high.
These are numbers i've debugged, the server pushes the player 8 tiles when mining anything and the space vacuum is always 12 or above.

Reply:
I'm just saying I've changed how space wind works in the other PR

Reply:
then merge your PR so i can make my changes or merge this first and change it on your end if the review process is going to be long

Reply:
Someone needs to Approve it though